[wrangler] Explain missing Flagship app ID (#14797)
cloudflare/workers-sdk: .changeset/tidy-flags-connect.md

---
"wrangler": patch
---

> Explain how to create draft Flagship bindings during remote development

Reviewer: ```suggestion
Explain how to provision Flagship bindings if `app_id` missing in remote development
```
